stackslib/src/chainstate/stacks/boot/docs.rs

```diff
@@ -33,7 +33,7 @@ This is the self-service interface.  tx-sender will be the Stacker.
 * You will need the minimum uSTX threshold. This isn't determined until the reward cycle begins, but this
    method still requires stacking over the _absolute minimum_ amount, which can be obtained by calling `get-stacking-minimum`.
 * The pox-addr argument must represent a valid reward address.  Right now, this must be a Bitcoin
-p2pkh or p2sh address.  It cannot be a native Segwit address, but it may be a p2wpkh-p2sh or p2wsh-p2sh address.
+p2pkh or p2sh address.  // Native Segwit addresses are now supported as of Stacks 2.1. but it may be a p2wpkh-p2sh or p2wsh-p2sh address.
```

Comment:
Thank you! Could you fix this so that the sentence syntax is correct in English?

Reply:
It has been fixed.